Framework/PythonInterface/mantid/simpleapi.py

```diff
@@ -983,6 +1008,14 @@ def __call__(self, *args, **kwargs):
                 If both startProgress and endProgress are supplied they will
                 be used.
                 """
+
+                # Check at runtime whether to throw upon alias deprecation.
+                if self._alias and self._alias.deprecated:
+                    deprecated = parse_date(self._alias.deprecated) < datetime.datetime.today()
+                    deprecated_action = ConfigService.Instance().get('algorithms.alias.deprecated', 'Warn').lower()
```

Comment:
Even you want it all lowercase!

Reply:
I'm allowing for a user forgetting to capitalize the initial 🙂